client/components/view-more-list/example.md

```diff
@@ -0,0 +1,9 @@
+```jsx
```

Comment:
This example should also be [added to `devdocs/examples.json`](https://github.com/woocommerce/wc-admin/blob/master/client/devdocs/examples.json), so it will display on the devdocs page - also if you run `npm run docs`, that will build the `/docs` markdown file.